src/include/ftp-server/dialogs.rb

```diff
@@ -121,26 +121,22 @@ def StartMode
       )
       Ops.set(
         result,
-        "get_service_start_via_xinetd",
+        "get_service_start_via_socket",
         fun_ref(method(:"started_via_socket?"), "boolean ()")
       )
       Ops.set(
         result,
-        "set_service_start_via_xinetd",
+        "set_service_start_via_socket",
         fun_ref(method(:"start_via_socket="), "void (boolean)")
       )
-      # TRANSLATORS: Radio selection
-      Ops.set(result, "start_auto_button", _("&When booting"))
-      Ops.set(result, "start_manual_button", _("&Manually"))
-      Ops.set(result, "start_xinetd_button", _("Via &xinetd"))
       Ops.set(
         result,
         "help",
         Builtins.sformat(
-          CWMServiceStart.AutoStartHelpXinetdTemplate,
-          _("When Booting"),
+          CWMServiceStart.AutoStartHelpSocketTemplate,
+          _("During Boot"),
           _("Manually"),
-          _("Via Socket")
+          _("Via socket")
```

Comment:
I think if we now use default we should use respective method https://github.com/teclator/yast-yast2/blob/76e0a3280dee002b4b62455eba0f2d35e5b1c2e1/library/cwm/src/modules/CWMServiceStart.rb#L263

this way translators won't need to translate this in different domain.